### ISSUE TITLE
Publish statically linked version

### ISSUE THREAD

**Issue by lpil** [stoeffel/elmi-to-json#23]
An issue to track the releasing of 0.19.2

Linux distros binary execution tested on:

- [x] alpine 
- [x] ubuntu
- [x] debian
- [x] fedora
- [x] centos

Is there anything else to do before release? :)

**Comment by stoeffel** [comment]
It didn't work on my friends machine. See https://github.com/stoeffel/elmi-to-json/pull/22#issuecomment-470010173

**Comment by lpil** [comment]
@stoeffel Can you provide any information on the machine please so we can replicate the problem? :)

**Comment by stoeffel** [comment]
@michaelerne would be awesome if you could provide some additional feedback on your machine. (do you remember the command I had you to run on your machine)? thanks ❤️

**Comment by michaelerne** [comment]
My machine runs Ubuntu 18.10. The problematic things seems to be my `LC_CTYPE` env var.

To reproduce, build the following Dockerfile:

```
FROM ubuntu:18.10
ADD https://github.com/stoeffel/elmi-to-json/releases/download/0.19.2-rc/elmi-to-json-0.19.2-rc-linux.tar.gz /

RUN mkdir -p /usr/local/bin \
 && tar -C /usr/local/bin -xzf /elmi-to-json*.tar.gz \
 && rm -f /elmi-to-json*.tar.gz

RUN apt-get update \
 && apt-get install -y language-pack-en \
 && rm -rf /var/lib/apt/lists/*
```

build:
```
docker build -t elmi-to-json:latest .
```

This fails:
```
# docker run --rm -ti elmi-to-json bash -c "LC_CTYPE=en_GB.UTF-8 elmi-to-json --help"
elmi-to-json: loadlocale.c:129: _nl_intern_locale_data: Assertion `cnt < (sizeof (_nl_value_type_LC_TIME) / sizeof (_nl_value_type_LC_TIME[0]))' failed.

```

This works:
```
# docker run --rm -ti elmi-to-json:latest bash -c "LC_CTYPE=C elmi-to-json --help"
elmi-to-json - Convert the interface info into json.

Usage: elmi-to-json [MODULE_PATHS] [-o|--output ARG]
  Get info for specific modules.

Available options:
  -h,--help                Show this help text
  MODULE_PATHS             Get info for specific modules.
  -o,--output ARG          Output info to a file.
```

The `LC_CTYPE` is the default set by Ubuntu if you select the UK as your region afaik. I did not deliberately set it myself. 


`elmi-to-json` seems to fail with other values as well, e.g. `en_US.UTF-8`,  `de_DE.UTF-8` and `de_CH.UTF-8` (for the last two i added `language-pack-de` to the Dockerfile`).

Hope that helps. Let me know if you need more info, happy to help.

**Comment by lpil** [comment]
That's wonderful, thank you very much!

**Comment by stoeffel** [comment]
@lpil it's still failing. I've tried it as well with the docker file described above.

**Comment by lpil** [comment]
The docker file above doesn't use the version with the fix, could that be the problem? Just want to be sure :)

**Comment by stoeffel** [comment]
Sorry, should have said that I changed the version and it still didn't work.

**Comment by lpil** [comment]
Thanks, I'll investigate this later :)

**Comment by bowsersenior** [comment]
Please have a look at https://github.com/stoeffel/elmi-to-json/pull/25 . Not quite sure how to test the binaries generated by Travis for the PR since they're not pushed to Github releases.

**Comment by stoeffel** [comment]
I just published 0.19.3

**Comment by lpil** [comment]
Great, thank you :)
